config.js

```diff
@@ -43,7 +43,8 @@ const ConfigOptions = () => {
          * Image-related variables
          */
         img: {
-            imgDir: 'img'
+            imgDir: 'img',
+            svgSpriteFilename: 'sprite.svg'
```

Comment:
I did wonder the same, then [came across this](https://english.stackexchange.com/questions/5366/which-is-correct-filename-file-name-or-filename). It's on the internet so it must be correct, right? 😜